pipelines/pipedrive/__init__.py

```diff
@@ -108,6 +118,8 @@ def _get_endpoint(entity, pipedrive_api_key, extra_params=None):
         params.update(extra_params)
     url = f'https://app.pipedrive.com/v1/{entity}'
     pages = _paginated_get(url, headers=headers, params=params)
+    if munge_custom_fields:
+        pages = map(functools.partial(munge_push_func if "Fields" in entity else pull_munge_func, endpoint=entity), pages)
```

Comment:
explain this code in a comment. what this `Field` means? this line is not self readable

Reply:
Sure, I'll do it.